### DISCUSSION TITLE
Temporary workaround to see the first chart without scrolling

### DISCUSSION THREAD

**Discussion by ludles** [breezy-weather/breezy-weather#783] [Show and tell]
Thanks to the new updates, I found a workaround to see most of the first chart without scrolling, as desired in https://github.com/breezy-weather/breezy-weather/issues/679#issuecomment-1949336565.

Comparing with GeometricWeather(right):
![Snipaste_2024-03-12_12-25-06](https://github.com/breezy-weather/breezy-weather/assets/37469405/7436a381-1b95-43a0-94fe-924031dfa3b6)

**Comment by papjul** [comment]
? What is the workaround?

**Reply by ludles** [reply]
I was planning to write it down in the first comment, but your reply was so fast. 😂

**Comment by ludles** [comment]
Method:
1. App version >= [v5.0.0](https://github.com/breezy-weather/breezy-weather/releases/latest);
2. Setting this app's DPI to 390 (**root required**).
    - I used [[MOD][XPOSED] App Settings Reborn](https://xdaforums.com/t/mod-xposed-app-settings-reborn.4141339/).
    - Changing the DPI also affects the display of widgets. So the parameter might need tuning depending on the device.

Thanks again for the devs' efforts in maintaining and improving this project! 💖
